plone/schemaeditor/browser/schema/add_field.py

```diff
@@ -47,7 +47,7 @@ def create(self, data):
 
         # remove fieldset_id from data
         if "fieldset_id" in data:
-            fieldset_id = data.pop("fieldset_id")
+            _ = data.pop("fieldset_id")
```

Comment:
The whole `if` statement could be replaced by `data.pop("fieldset_id", None)`. But we don't have to do it in this PR. Keeping the manual changes small is good.